docs/getting_started.md

```diff
@@ -0,0 +1,124 @@
+                  "eks:DescribeCluster",
+                  "eks:DescribeFargateProfile",
+                  "eks:DescribeIdentityProviderConfig",
+                  "eks:DescribeNodegroup",
+                  "eks:DescribeUpdate",
+                  "eks:ListAddons",
+                  "eks:ListClusters",
+                  "eks:ListFargateProfiles",
+                  "eks:ListIdentityProviderConfigs",
+                  "eks:ListNodegroups",
+                "eks:ListTagsForResource",
+                "eks:ListUpdates"
+            ],
+            "Resource": [
+                "arn:aws:eks:*:844333597536:identityproviderconfig/*/*/*/*",
+                "arn:aws:eks:*:844333597536:fargateprofile/*/*/*",
+                "arn:aws:eks:*:844333597536:nodegroup/*/*/*",
+                "arn:aws:eks:*:844333597536:cluster/*",
+                "arn:aws:eks:*:844333597536:addon/*/*/*"
+            ]
+        }
+    ]
+}
+EOF
+aws iam create-policy --policy-name kyverno-aws-adapter-policy --policy-document file://my-policy.json
+```
+
+**Note:** Make sure your AWS CLI is configured correctly. Follow the [official guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html) for setting it up
+
+
+### Creating IAM OIDC provider
+An IAM Open ID Connect provider for the cluster is required to provide the reference in the IAM Role.
+
+```bash
+eksctl utils associate-iam-oidc-provider --cluster <cluster-name> --approve --region <region-code>
+```
+
+### Creating the IAM Role
+Create an IAM Role that references the policy we created earlier.
+
+**Note:** We will use `eksctl` in this guide to create the IAM Role. The serviceaccount creation will be done by the Helm chart so that appropriate RBAC is assigned by the chart itself. If you wish to use your own serviceaccount, then make sure all the necessary rolebinding and clusterrolebinding are present.
+
+```bash
+account_id=$(aws sts get-caller-identity --query "Account" --output text)
+
+eksctl create iamserviceaccount --name nirmata-aws-adapter  --namespace nirmata-aws-adapter --cluster <cluster-name> --role-name nirmata-adapter-role --attach-policy-arn arn:aws:iam::$account_id:policy/kyverno-aws-adapter-policy   --role-only  --approve
+```
+
+This will create a new IAM Role that references the policy we created above. This also creates the trust-relationship for us. If you wish to create the IAM Role via the management console or the AWS CLI, make sure to create the trust-relationship so that the AWS Adapter can assume this Role to fetch EKS Cluster details.
+
+
+### Installing the AWS Adapter Helm chart
+First we need to set the variables that are needed to install the chart. You can either pass the arguments via the command line or set them in a file.
+
+Here is the minimal info that you need to set
+```bash
+> cat myvalues.yaml
+eksCluster:
+  name: cluster-name
+  region: cluster-region
+
+roleArn: arn:aws:iam::account-id:role/nirmata-adapter-role
```

Comment:
A placeholder value seems more appropriate since we're showing a sample values file here
```suggestion
roleArn: arn:aws:iam::111122223333:role/nirmata-adapter-role
```